src/installer/pkg/projects/Directory.Build.targets

```diff
@@ -14,6 +14,17 @@
     <StableVersion Condition="'$(IncludePreReleaseLabelInPackageVersion)' != 'true'">$(Version)</StableVersion>
   </PropertyGroup>
 
+  <!--
+    For mobile targets, we do not need the host, so allow depproj's to opt 
+    in in order to skip building. 
+  -->
+  <Target Name="MobileSkipBuildProps" BeforeTargets="GetSkipBuildProps" 
+    Condition="'$(SkipBuildOnRuntimePackOnlyOS)' and '$(TargetOS)' == 'iOS'">
```

Comment:
Nit, we usually not use boolean conditions based from properties, we usually do direct string comparison ourselves, because boolean condition, forces you to set it to either false or true in all code paths. If we made this a boolean, the default could be false, so sometime like `'$(SkipBuildOnRuntimePackOnlyOS)' != 'true'` then that way you only set it to true when you want to disable this target but not set it to false anywhere.